dev-docs/source/SystemTests.rst

```diff
@@ -236,6 +236,32 @@ properties, go to ```Command Arguments``` and append flags as appropriate.
 For example, adding ``-R ISIS`` will run any tests which match the regular
 expression ``ISIS``.
 
+Debugging System Tests in Pycharm
+---------------------------------
+
+System tests can be debugged from Pycharm without finding and attaching to
+the process, or using a remote debugger.
+
+To do this, create a new Configuration (Run -> Edit Configurations), and add
+a new Python configuration with the script path set to ``runSystemtests.py``
+This is found in ``/Testing/SystemTests/Scripts/runSystemTests.py``
+
+The parameters for the configuration can be set just like the command line
+args when running the tests from the ``systemtest.bat`` script, e.g pass
```

Comment:
On macOS/Linux the script is unfortunately called `systemtest` so could we say `systemtest.bat/systemtest` in script to be clear this is not Windows only?